modules/nw-cudn-cr.adoc

```diff
@@ -0,0 +1,169 @@
+//module included in the following assembly:
+//
+// * networking/multiple_networks/primary_networks/about-user-defined-networks.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-cudn-cr_{context}"]
+= Creating a ClusterUserDefinedNetwork custom resource
+
+The following procedure creates a `ClusterUserDefinedResource` custom resource definition (CRD). Based upon your use case, create your request using either the `cluster-layer-two-udn.yaml` example for a `Layer2` topology type or the `cluster-layer-three-udn.yaml` example for a `Layer3` topology type.
+
+[IMPORTANT]
+====
+The `ClusterUserDefinedNetork` CRD is intended for use by cluster administrators and should not be used by non-administrators. If used incorrectly, it might result in security issues with your deployment, cause disruptions, or break the cluster network. 
```

Comment:
```suggestion
The `ClusterUserDefinedNetwork` CRD is intended for use by cluster administrators and should not be used by non-administrators. If used incorrectly, it might result in security issues with your deployment, cause disruptions, or break the cluster network. 
```

Reply:
fixed.